src/components/base/base.ts

```diff
@@ -51,7 +51,7 @@ export interface Base extends Resource {
   stop(extra?: Struct): Promise<void>;
 
   /** Return true if the base is in motion. */
-  isMoving(extra?: Struct): Promise<boolean>;
+  isMoving(): Promise<boolean>;
```

Comment:
This is technically a breaking change. None of the other `isMoving` functions have an `extra` parameter, so I thought to remove it. Let me know if I should not do this.